src/Modules/CalcPerform.lua

```diff
@@ -2853,6 +2863,65 @@ function calcs.perform(env, avoidCache)
 		end
 	end
 
+	-- Triggered by parent attack
+	if env.minion and env.player.mainSkill.minion then
+		if env.minion.mainSkill.skillData.triggeredByParentAttack then
+			local spellCount = {}
+			local trigRate = 0
+			local source = nil
+			for _, skill in ipairs(env.player.activeSkillList) do
+				if skill.skillTypes[SkillType.Attack] and skill ~= env.player.mainSkill then
+					source, trigRate = findTriggerSkill(env, skill, source, trigRate)
+				end
+			end
+			
+			local icdr = calcLib.mod(env.minion.mainSkill.skillModList, env.minion.mainSkill.skillCfg, "CooldownRecovery")
+			t_insert(spellCount, { uuid = cacheSkillUUID(env.minion.mainSkill), cd = env.minion.mainSkill.skillData.cooldown / icdr, next_trig = 0, count = 0 })
+
+			if not source then
+				env.minion.mainSkill.skillData.triggeredByParentAttack = nil
+				env.minion.mainSkill.infoMessage = "No triggering Skill Found"
+				env.minion.mainSkill.infoMessage2 = "DPS reported assuming regular cast"
+				env.minion.mainSkill.infoTrigger = ""
+			else
+				env.minion.mainSkill.skillData.triggered = true
+				local uuid = cacheSkillUUID(source)
+
+				local sourceAPS = GlobalCache.cachedData["CACHE"][uuid].Speed
+
+				-- Get action trigger rate
+				-- Ugly hack to get data to go into minion actor instead of player
+				local breakdownBak = breakdown
+				local outputBak = output
+				breakdown = env.minion.breakdown
+				output = env.minion.output
+
+				trigRate = calcActualTriggerRate(env, source, sourceAPS, spellCount, output, breakdown, false, true)
```

Comment:
Is there a reason:
`trigRate = calcActualTriggerRate(env, source, sourceAPS, spellCount, output, breakdown, false, true)`
isn't 
`trigRate = calcActualTriggerRate(env, source, sourceAPS, spellCount, env.minion.output, env.minion.breakdown, false, true)`?

That should work and eliminate the need for the "ugly hack".